motr/setup.c

```diff
@@ -82,20 +82,19 @@ extern struct m0_reqh_service_type m0_ss_svc_type;
  * special "safety" coefficient is introduced to increase space in repair zone.
  * Safety coefficient is defined as "safety mul"/"safety div".
  *
- * EOS-23180: Previous safety coefficent[3/2] was causing issues and
- * repair zone was occupying more space than normal zone. Details are in the bug.
- * Getting rid of the safety coefficient. Also adding an ASSERT that
- * repair zone percentage should not cross M0_BC_REPAIR_ZONE_MAX_ALLOWED percent.
- * User has to make sure repair zone percentage should not cross this by
- * choosing appropriate spare value.
+ * If repair zone percentage crosses M0_BC_REPAIR_ZONE_MAX_ALLOWED,
+ * user has to choose appropriate spare value.
```

Comment:
To catch this issue in EC config stage itself, can you create a hare improvement/story ticket to have this formula and assert if more than 20% spare space reservation is done?

Reply:
Created Madhav. 
https://jts.seagate.com/browse/EOS-23414